Modules/Core/Common/test/itkShapedNeighborhoodIteratorTest.cxx

```diff
@@ -191,6 +192,21 @@ itkShapedNeighborhoodIteratorTest(int, char *[])
   }
   std::cout << it.GetPixel(off) << std::endl;
 
+  println("Testing interoperability with FlatStructuringElement");
+  using SE2Type = itk::FlatStructuringElement<4>;
+  SE2Type::RadiusType r;
+  r.Fill(2);
+  SE2Type k;
+  k = SE2Type::Cross(r);
```

Comment:
Please consider initialization, instead of declaration + assignment, for those two local variables:

```cpp
    const SE2Type::RadiusType r = SE2Type::RadiusType::Filled(2);
    const SE2Type k = SE2Type::Cross(r);
```
You may then even declare them `const`. 😺

Reply:
I was concerned with things working, I totally forgot about style 😄